web/src/main/java/org/openmrs/web/filter/util/CustomResourceLoader.java

```diff
@@ -92,7 +92,8 @@ private Locale parseLocaleFrom(String filename, String basename) {
 		Locale result = null;
 		
 		if (filename.startsWith(basename)) {
-			filename = filename.substring(basename.length());
+			String tempfilename = filename.substring(basename.length());
+			filename = tempfilename;
```

Comment:
In the issue it says "Background Information: **Method parameters**, caught exceptions and foreach variables **should not be reassigned**". Here the `filename` is a method parameter. So you should assign the method parameter to a temporary variable and reuse it there after.

Reply:
The last two commit only adds a new line :)